test/test-oauth.js

```diff
@@ -36,7 +36,7 @@ describe('OAuth 2.0', () => {
 					callback({ url: `http://localhost/Callback?access_token=997f871ee3194eb39b4f2a19d5871cb8608e9c0a&state=${state}&token_type=Bearer` });
 				});
 			};
-			OAuth.authorizeImplicitly(implicit.url, implicit.clientId, function (err, oauth) {
+			OAuth.authorizeImplicitly(implicit.url, implicit.clientId, 'scope', function (err, oauth) {
```

Comment:
I'd suggest retaining the existing tests as-is and adding tests that also pass in scopes.